docs/develop/make-office-web-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -0,0 +1,85 @@
+
+You can configure your Office web add-in so that when an equivalent COM add-in is already installed on the user's computer, Office runs the COM add-in instead of your web add-in. The COM add-in is called "equivalent" because Office will seamlessly transition between the COM add-in and the web add-in depending on which is installed on Windows.
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+## Specify an equivalent COM add-in in the manifest
+
+To enable compatibility with an existing COM add-in, identify the equivalent COM add-in in the manifest of your web add-in. Then Office will use the COM add-in instead of your web add-in when running on Windows.
+
+Specify the `ProgID` of the equivalent COM add-in. Office will then use the COM add-in UI instead of your web add-in's UI when the COM add-in is installed.
+
+The following example shows how to specify both a COM add-in and an XLL as equivalent. Often you will specify both so for completeness this example shows both in context. They are identified by their `ProgID` and `FileName` respectively. For more information on XLL compatibility, see [Make your custom functions compatible with XLL user-defined functions](../excel/make-custom-functions-compatible-with-xll-udf.md).
+
+```xml
+<VersionOverrides>
+...
+<EquivalentAddins>
+  <EquivalentAddin>
+    <ProgID>ContosoCOMAddin</ProgID>
+    <Type>COM</Type>
+  </EquivalentAddin>
+
+  <EquivalentAddin>
+    <FileName>contosofunctions.xll</FileName>
+    <Type>XLL</Type>
+  </EquivalentAddin>
+<EquivalentAddins>
+...
+</VersionOverrides>
+```
+
+## Equivalent behavior for users
+
+When an equivalent COM add-in is specified in the web add-in manifest, Office suppresses your web add-in's UI on Windows when the equivalent COM add-in is installed. This does not affect your web add-in's UI on other platforms like online or macOS. Office only hides the ribbon buttons and does not prevent installation. Therefore your web add-in will still appear in the following UI locations:
+
+- Under **My Add-ins** because it is technically installed.
+- As an entry in the ribbon manager.
+
+The following scenarios describe what happens depending on how the user acquires the web add-in.
+
+### Office Store acquisition of web add-in
+
+If a user downloads the web add-in from the Office store, and the equivalent COM add-in is already installed, then Office will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Centralized deployment of web add-in
+
+If an admin deploys the web add-in to their tenant using centralized deployment, and the equivalent COM add-in is already installed, the user needs to restart Office before they will see any changes. After they restart Office, Office will:
```

Comment:
```suggestion
If an admin deploys the web add-in to their tenant using centralized deployment, and the equivalent COM add-in is already installed, the user needs to restart Office before they will see any changes. After Office restarts, it will:
```